src/trex/utils.py

```diff
@@ -13,3 +18,42 @@ def format(self, record):
         if record.levelno != logging.INFO:
             record.msg = "{}: {}".format(record.levelname, record.msg)
         return super().format(record)
+
+
+def molecule_list_2_df(mol_list: List[Molecule]) -> pd.DataFrame:
```

Comment:
`mol_list` can be named `molecules`

I try to follow this naming convention:

`item`: An instance of a class named `Item`
`items`: Something containing multiple `Item` instances (list, tuple, iterator)